pymc3/distributions/multivariate.py

```diff
@@ -690,6 +692,143 @@ def logp(self, x):
         )
 
 
+class DirichletMultinomial(Discrete):
+    R"""Dirichlet Multinomial log-likelihood.
+
+    Dirichlet mixture of multinomials distribution, with a marginalized PMF.
+
```

Comment:
We should add a link to the example NB by @bsmith89 here (note that it's not merged yet)

Reply:
Not sure what the usual policy is, but I think merging this PR first, and then adding a note when the example is ready might be the best option.

Reply:
Yep, sounds good 👍 Just wanted to make sure the example NB was on everyone's radar and that we didn't forget adding it once it's merged 😉